lib/client.js

```diff
@@ -189,6 +189,60 @@ class GoogleOauth2Client {
     return result;
   }
 
+  async writeColumnToSpreadsheet(opts) {
+    this.logger.debug('Trying to write a column to spreadsheet...');
+    const { spreadsheetId } = this.configuration;
+    const { range, values } = opts;
+    const sheets = google.sheets({ version: sheetsVersion, auth: this.client });
+    const result = await sheets.spreadsheets.values.append({
+      spreadsheetId,
+      range,
+      valueInputOption: 'RAW',
+      resource: {
+        majorDimension: 'COLUMNS',
+        values: [values],
+      },
+    });
+    this.logger.debug('Spreadsheet column written successfully');
+    return result;
+  }
+
+  async updateSpreadsheetRow(opts) {
+    this.logger.debug('Trying to update a row in spreadsheet...');
+    const { spreadsheetId } = this.configuration;
+    const { range, values } = opts;
+    const sheets = google.sheets({ version: sheetsVersion, auth: this.client });
+    const result = await sheets.spreadsheets.values.update({
+      spreadsheetId,
+      range,
+      valueInputOption: 'RAW',
+      resource: {
+        majorDimension: 'ROWS',
+        values: [values],
+      },
+    });
+    this.logger.debug('Spreadsheet row updated successfully');
+    return result;
+  }
```

Comment:
It's basically same as `writeColumnToSpreadsheet` with small difference - `majorDimension` and log messages, can be merged in one

Reply:
no, for _updateSpreadsheetRow_ it is used `update` spreadsheet function and for _writeColumnToSpreadsheet_ - `append`

Reply:
merged updateSpreadsheetRow with updateSpreadsheetColumn and writeColumnToSpreadsheet with writeToSpreadsheet